README.md

```diff
@@ -219,6 +220,97 @@ The library adds a `form-submitted` to the submitted form. You can use it to sty
   }
 }
 ```
+- `controlErrorComponent` - Optional. Allows changing the default component that is used to render 
+  the errors. This component should implement the `ControlErrorComponent` interface. If you only need to
+  replace the error component's template, you may derive it from the default component, 
+  `DefaultControlErrorComponent`, and provide the requisite HTML template.
+
+  A common example is when using Ionic forms where each form field is wrapped in an `ion-item` and errors
+  are best displayed as a sibling `ion-item` of the field. Example below shows how this can be done using 
+  a custom control error component.
+
+  For example:
+  ```ts
+  // Custom error component that will replace the standard DefaultControlErrorComponent.
+  @Component({
+    template: `
+    <ion-item lines="none" class="ion-text-wrap" [class.hide-control]="hide">
```

Comment:
Let's make the public properties more readable for the consumers:

hide => hideError
_tpl => errorTemplate
_text => errorText
context => errorContext

Reply:
Excellent idea. Small things like this can go a long way to ease the pain of using a library!